app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
   has_many :posts, foreign_key: 'author_id', primary_key: 'id'
   has_many :comments
   has_many :likes
```

Comment:
- [ ] 👋 Hey there! 👩‍💻. Great job on your PR! 🚀 As part of our project's user authentication requirements, we need to ensure email confirmation for our users.  To achieve this, it's essential to enable the `: confirmable` module in your user model. Your user mode should look like :

```
devise :database_authenticatable, :registerable,
				:recoverable, :rememberable, :validatable, :confirmable

```

#### Happy coding! 😃👍